pkg/policies/opa/rego/docker/docker_run/AC_DOCKER_0052.json

```diff
@@ -0,0 +1,17 @@
+{
+    "name": "runusingupgrade",
+    "file": "runusingupgrade.rego",
+    "policy_type": "docker",
+    "resource_type": "run",
```

Comment:
I think `resource_type` is supposed to be `docker_run` here.

@Rchanger could you please help in verification of this param's value?

Reply:
I feel its much better to use resource type as docker_run

Reply:
It should be run

Reply:
@Rchanger 

For azure, aws, gcp, and k8s, the policy team is following the convention mentioned.

So, could we continue the same with Docker policies as well?

Reply:
Yes you are right I will make the changes in my provider code as well

Reply:
@pavniii 

Are you working on these changes? Please update your status.